### PR TITLE
docs: boolean value in docker-compose examples not deserializable in env variables

### PR DESCRIPTION
**Pull request by ursuscamp**
## What type of PR is this?

- documentation

## What this PR does / why we need it:

Booleans values in docker-compose environment variables are not deserializable and results in an error like this

```
ERROR: The Compose file './docker-compose.yaml' is invalid because:
services.mealie.environment.ALLOW_SIGNUP contains false, which is an invalid type, it should be a string, number, or a null
```

## Which issue(s) this PR fixes:

N/A

### CODE REVIEW

**Comment by michael-genson** [comment]
Are you using `docker compose` or `docker-compose`? The latter is deprecated.

Otherwise, make sure docker is up to date.

**Review by michael-genson — CHANGES_REQUESTED** [review]
Thanks! I noticed it works fine in mine without quotes, but I think it's because I use list syntax instead of map syntax? Maybe it's undefined behavior? Who knows, I can barely find docs about it.

Would we be able to use double quotes? Only because we already have the port in double quotes.

File: docs/docs/documentation/getting-started/installation/postgres.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ services:
       - mealie-data:/app/data/
     environment:
       # Set Backend ENV Variables Here
-      ALLOW_SIGNUP: false
+      ALLOW_SIGNUP: 'false'
```
Comment (hay-kot) [inline docs/docs/documentation/getting-started/installation/postgres.md]:
```suggestion
      ALLOW_SIGNUP: "false"
```

File: docs/docs/documentation/getting-started/installation/sqlite.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ services:
       - mealie-data:/app/data/
     environment:
       # Set Backend ENV Variables Here
-      ALLOW_SIGNUP: false
+      ALLOW_SIGNUP: 'false'
```
Comment (hay-kot) [inline docs/docs/documentation/getting-started/installation/sqlite.md]:
```suggestion
      ALLOW_SIGNUP: "false"
```

**Review by hay-kot — CHANGES_REQUESTED** [review]

**Review by hay-kot — APPROVED** [review]

**Review by michael-genson — APPROVED** [review]

**Pull request merged**